packages/account/src/Sections/Security/Passkeys/components/__tests__/passkey-modal.spec.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import PasskeyModal from '../passkey-modal';
+
+describe('PasskeyModal', () => {
+    let modal_root_el: HTMLElement;
+    beforeAll(() => {
+        modal_root_el = document.createElement('div');
+        modal_root_el.setAttribute('id', 'modal_root');
+        document.body.appendChild(modal_root_el);
+    });
+
+    afterAll(() => {
+        document.body.removeChild(modal_root_el);
+    });
+
+    it('renders the modal correctly and responds to user interaction', () => {
+        const handleClick = jest.fn();
```

Comment:
Maybe we can move this mockfunction outide and reuse it

Reply:
yes, we can. But it will just increase the quantity of lines in our case.
because we will also have to clean mocks before each test.
so here having two separate variables is just shorter :)